public/app/features/explore/state/main.test.ts

```diff
@@ -164,6 +165,29 @@ describe('Explore reducer', () => {
           .thenStateShouldEqual({
             left: leftItemMock,
             right: undefined,
+            syncedTimes: false,
+          } as unknown as ExploreState);
+      });
+
+      it('should unsync time ranges', () => {
+        const itemMock = {
+          containerWidth: 100,
+        } as unknown as ExploreItemState;
+
+        const initialState = {
+          left: itemMock,
+          right: itemMock,
+          syncedTimes: true,
+        } as unknown as ExploreState;
+
+        reducerTester<ExploreState>()
+          .givenReducer(exploreReducer, initialState)
+          .whenActionIsDispatched(splitCloseAction({ itemId: ExploreId.right }))
+          .thenStateShouldEqual({
+            evenSplitPanes: true,
```

Comment:
Should it be here? I think split resize introduced this property and wasn't backported to 9.3.x 🤔

Reply:
great catch! missed this when solving thew conflict!